torch/csrc/autograd/profiler.cpp

```diff
@@ -37,13 +35,15 @@ namespace {
     CUDA_RECORDED,
     CUDA_MEM_USAGE,
     CUDA_DEVICE,
-    CUDA_US
+    CUDA_US,
+    kEventIValuesSize // must be last in list
```

Comment:
could you keep the style of enum elements, e.g. ``NUM_EVENT_IVALUE_IDX`` and the same for below